yarn.lock

```diff
@@ -1626,7 +1626,49 @@ eslint-watch@^3.1.2:
     text-table "^0.2.0"
     unicons "0.0.3"
 
-eslint@^4.0.0, eslint@^4.6.1:
+eslint@^4.0.0:
```

Comment:
How you got an old version of eslint installed locally? Perhaps version `4.11.0`?